src/storage/src/table/batch_table/storage_table.rs

```diff
@@ -238,7 +241,29 @@ impl<S: StateStore> StorageTableInner<S, EitherSerde> {
         let row_serde = {
             let schema = Arc::from(data_types.into_boxed_slice());
             if versioned {
-                ColumnAwareSerde::new(&column_ids, schema).into()
+                let column_with_default = table_columns
```

Comment:
Is it possible to let `ColumnAwareSerde` take `table_columns` and handle this inside? So that we can DRY for both tables.

Reply:
My concern is that `build_from_prost` is in `risingwave_expr` while serdes are in `risingwave_common`. Can `risingwave_common` use sth in `risingwave_expr`?

Reply:
I'm afraid we can't (or better not to) 😰. What about moving the `ValueRowSerde` traits and all implementations to `storage` crate? This makes more sense as it's the storage layer's responsibility to erase the difference between column-not-present and column-is-default.

Reply:
This sounds like a non-negligible amount of work, but if it worths I'll do it.